core/src/core-api/ux.js

```diff
@@ -176,6 +176,18 @@ class LuigiUX {
   getDirtyStatus() {
     return Luigi.getDirtyStatus();
   }
+
+  /**
+   * Retrieves the current active theme. Falls back to **defaultTheme** if none explicitly specified before.
```

Comment:
```suggestion
   * Returns the current active theme. Falls back to **defaultTheme** if one wasn't explicitly specified before.
```
make it follow existing pattern from the previous description.